freqtrade/enums/signaltype.py

```diff
@@ -7,6 +7,8 @@ class SignalType(Enum):
     """
     BUY = "buy"
     SELL = "sell"
+    SHORT = "short"
+    EXIT_SHORT = "exit_short"
```

Comment:
I'd rather add required entries here as part of the "strategy-interface change" PR - as these additions are not correct based on the discussion in the interface change Issue.

Reply:
That should be fine, it would be helpful to keep in in case another PR needs to access it, so we don't have the changes across multiple PRs

Reply:
I'd like to keep it in this PR. Right now only the strategy interface needs it, but it could be useful to have in there. If you disagree could you just remove it before you merge.

Reply:
i think changes to this particular enum should be done when the referencing code is touched.

Right now, it's adding 2 unused values - while keeping the other 2 values identical (which is wrong, based on the desired future state).
As we'll need to have some "compat" code (ensuring compitibility of current strategies), we'll need to see if this enum will need 4 or 6 entries (keeping the existing ones for now).

So the reasoning (keep changes to one PR) will not hold.